simulation_parameters/Constants.cs

```diff
@@ -793,12 +793,12 @@ public static class Constants
     /// <summary>
     ///   The speed of which a cell can absorb compounds from digestible engulfed objects.
     /// </summary>
-    public const float ENGULF_COMPOUND_ABSORBING_PER_SECOND = 0.5f;
+    public const float ENGULF_COMPOUND_ABSORBING_PER_SECOND = 0.3f;
 
     /// <summary>
-    ///   How much compounds a cell can absorb per second from digestible engulfed objects.
+    ///   How much compounds in relation to real compound amount can be absorbed from digestible engulfed objects.
     /// </summary>
-    public const float ENGULF_BASE_COMPOUND_ABSORPTION_YIELD = 0.3f;
+    public const float ENGULF_BASE_COMPOUND_ABSORPTION_YIELD = 1.8f;
```

Comment:
Would the balance work if this was at 1? Lysosome is meant to further increase the absorption efficiency. I ask because we try to pretend things are physically realistic in Thrive, but here if we multiply the compound amount by 1.8 we are basically creating new matter out of nothing.

Reply:
True, but while testing prokaryotic, I got like up to 1 glucose from engulfing cell half my size which isn't much to suffice. One option is to give the stuff cell is built of more glucose, that would solve that problem.